apps/web-api/src/utils/base.service.ts

```diff
@@ -65,24 +69,56 @@ export class BaseService<T, S> {
       return;
     }
 
-    const whereCondition = {};
-    for (const [field, operators] of Object.entries<IWhereOperators>(
-      args.where,
-    )) {
-      const operatorNames = Object.keys(operators) as Array<
-        keyof IWhereOperators
-      >;
-
-      if (operatorNames.length !== 1) {
-        throw new Error(
-          'For every field you should use only one condition operator.',
+    const subConditions = [];
+    const generateWhereCondition = (whereOperators: TWhereParams<S>) => {
+      const whereCondition = {};
+      for (const [field, operators] of Object.entries<IWhereOperators>(
+        whereOperators,
+      )) {
+        const operatorNames = Object.keys(operators) as Array<
+          keyof IWhereOperators
+        >;
+
+        const operatorName = operatorNames[0];
+        const ormOperator = this.getOrmWhereOperator(operatorName);
+        const ormOperation = this.getOrmWhereOperation(field);
+
+        if (ormOperator) {
+          whereCondition[field] = ormOperator(operators[operatorName]);
+        } else if (ormOperation) {
+          subConditions.push(generateWhereCondition(whereOperators[field]));
+        } else {
+          throw new Error(`Unknown GQL condition operator '${operatorName}'.`);
+        }
+      }
+      return whereCondition;
+    };
+
+    const condition = generateWhereCondition(args.where);
+    qb.andWhere(condition);
+    this.applySubWhere(qb, subConditions);
+  }
+
+  private applySubWhere(
+    qb: SelectQueryBuilder<T>,
+    conditions: TWhereCondition[],
+  ) {
+    if (!conditions.length) return;
```

Comment:
Please use curly braces({}) for all if statements, without exception for early exit.

Reply:
Maybe we should add special rule to linter config.